index.html

```diff
@@ -346,6 +349,40 @@ <h1 class="section-header fadeIn-scroll">Schedule</h1>
                 </div>
             </section>
             
+            <!-- Hack Frost 1.0 -->
+            <section class="schedule" id="hackfrost1">
+               <div class="wrapper">
+               <h1 class="section-header fadeIn-scroll">Hack Frost 1.0</h1>
```

Comment:
For the sake of consistency with the event, can we change this to 
```suggestion
               <h1 class="section-header fadeIn-scroll">Hack Frost NL 1.0</h1>
```